pkg/apis/provisioning/v1alpha5/requirements.go

```diff
@@ -78,6 +78,7 @@ var (
 	// and prevent validation errors when specified
 	IgnoredLabels = stringsets.NewString(
 		v1.LabelTopologyRegion,
+		v1.LabelFailureDomainBetaRegion,
```

Comment:
Check out line 75 -- this is already a normalized label. WDYT about normalizing it and then ignoring it to avoid repeating ourselves in this ignored list?

Reply:
The filtering is before normalization, this way it is more efficient by skipping unnecessary normalization.
The only reason that I am keeping the label in the normalization label list is that in case in the future we want to allow the region label for some reason, we won't forget to normalize it.

Reply:
The normalization is effectively free. It's just a single map lookup, and only improves performance if the label is ignored. I'd much rather maintain the separation of concerns of "region is ignored" and "there two multiple values that normalize to region".

This can be trivially implemented by swapping

```
		if IgnoredLabels.Has(requirement.Key) {
			continue
		}
		if normalized, ok := NormalizedLabels[requirement.Key]; ok {
			requirement.Key = normalized
		}
```
to be

```
		if normalized, ok := NormalizedLabels[requirement.Key]; ok {
			requirement.Key = normalized
		}
		if IgnoredLabels.Has(requirement.Key) {
			continue
		}
```